lib/rspec-gc-control/example.rb

```diff
@@ -7,18 +7,28 @@ module Core
     class Example
     private
 
-      undef :record_finished
-      def record_finished(status, results={})
-        finished_at = Time.now
-        record results.merge(:status => status, :finished_at => finished_at, :run_time => (finished_at - (execution_result[:started_at] + (execution_result[:gc_time] || 0))))
-      end
+      if defined?(::RSpec::Core) && ::RSpec::Core::Version::STRING >= '3.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.